src/loaders/UrlNode.js

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
this seems like duplicate, how do we sync the TS and JS versions?

What errors does rollup throw?

Reply:
A warning, not an error.

<img width="1264" alt="Screen Shot 2022-09-15 at 11 48 30" src="https://user-images.githubusercontent.com/1946920/190324416-3ef9924e-1884-4e7e-b926-cb1cd7d8793b.png">

But I'm 100% sure that you or [pavandv](https://github.com/pavandv) waste several hours removing it if saw this thing.
Just awaiting TS migration and will remove it then.

Reply:
I suggest closing eyes for this now and merge as is. We clean up this after migration on NX + typescript. Moreover gitlab actions bot already opened an issue according to TODO in the code: https://github.com/module-federation/nextjs-mf/issues/167

One **js**-UrlNode for build (webpack) side.
Another **ts**-UrlNode for runtime in MFClient. So client bundle has just one copy of UrlNode.